autogen/main/masq.tf.tmpl

```diff
@@ -19,7 +19,7 @@
 /******************************************
   Create ip-masq-agent confimap
  *****************************************/
-resource "kubernetes_config_map" "ip-masq-agent" {
+resource "kubernetes_config_map" "ip_masq_agent" {
```

Comment:
Hey @wyardley - I don't see a moved for `ip_masq_agent`?

Reply:
Good call, I probably missed that. Thanks for checking.

Reply:
Updated